problem_builder/mentoring.py

```diff
@@ -63,6 +64,10 @@
     'locations': ['public/themes/lms.css']
```

Comment:
@itsjeyd Is this `_default_theme_config` being used? Looks to me like with `ThemableXBlockMixin` you need to define `default_theme_config` as an attribute on the class instead.